packages/@react-types/searchwithin/package.json

```diff
@@ -0,0 +1,20 @@
+{
+  "name": "@react-types/searchwithin",
+  "version": "3.1.1",
```

Comment:
This should be 3.0.0-alpha.1 right? Needs `"private": true` as well

Reply:
thanks, good catch on the versioning